tf_encrypted/convert/convert.py

```diff
@@ -95,6 +95,18 @@ def convert(
           continue
 
         self.outputs[output] = register[node.op](self, node, inputs)
```

Comment:
Shouldn't this line be deleted now?

Reply:
Ah yes, my bad, it should have been removed.